pageserver/src/tenant_mgr.rs

```diff
@@ -75,9 +76,13 @@ pub fn load_local_repo(
         // Set up a WAL redo manager, for applying WAL records.
         let walredo_mgr = PostgresRedoManager::new(conf, tenant_id);
 
+        // Load config file
+        let tenant_conf = LayeredRepository::load_tenant_config(conf, tenant_id).unwrap();
```

Comment:
Lets use a result here instead of a panic. Because this function is used in `apply_timeline_sync_status_updates` so if it panics this will affect other tenants